src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -137,7 +138,7 @@ std::shared_ptr<ov::Node> make_ov_reduction_op(const Node& node,
     if (reduction_axes != nullptr) {
         return std::make_shared<OpType>(ov_input, reduction_axes, static_cast<bool>(keepdims));
     } else {
-        return ai_onnx::opset_1::identity(node).at(0).get_node_shared_ptr();
+        return std::make_shared<v16::Identity>(ov_input);
```

Comment:
Why this change is needed? We do generate `Identity` node in `ai_onnx::opset_1::identity(node)`

Reply:
The `node` is `Exp`'s input. But the `Identity` want to repleace the `Reduce`.